.env

```diff
@@ -34,7 +32,7 @@ APP_XDEBUG_HOST=docker.for.mac.localhost
 
 ###> snc/redis-bundle ###
 # passwords that contain special characters (@, %, :, +) must be urlencoded
-REDIS_URL=redis://redis
+REDIS_HOST=redis
```

Comment:
Is already present in line 17. why this change?

Reply:
just remove REDIS_URL